src/CodeGen_X86.cpp

```diff
@@ -220,33 +230,6 @@ void CodeGen_X86::visit(const Sub *op) {
     }
 }
 
-void CodeGen_X86::visit(const Mul *op) {
-
-#if LLVM_VERSION < 110
```

Comment:
Do we know that this is no longer necessary? (We haven't yet dropped support for LLVM10)

Reply:
I didn't delete it, it's moved to the Call visitor.